app/controllers/stacks_controller.rb

```diff
@@ -15,7 +15,9 @@ def show
     @stack = Stack.from_param!(params[:id])
     return if flash.empty? && !stale?(last_modified: @stack.updated_at)
 
-    @tasks = @stack.tasks.order(id: :desc).preload(:since_commit, :until_commit, :user).limit(10)
+    ntasks = params[:ntasks].to_i || 10
```

Comment:
- `nil.to_i || 10 #=> nil`
- Can you add tests please?
- Can you call this `task_count` or something more obvious?